orc8r/cloud/go/services/tenants/obsidian/handlers/handlers.go

```diff
@@ -140,11 +147,65 @@ func DeleteTenantHandler(c echo.Context) error {
 		return terr
 	}
 	err := tenants.DeleteTenant(c.Request().Context(), tenantID)
+	err = errorHandlingForGet(err, fmt.Errorf("Tenant %d does not exist", tenantID), err)
+	if err != nil {
+		return err
+	}
+	return c.NoContent(http.StatusNoContent)
+}
+
+func GetControlProxyHandler(c echo.Context) error {
+	tenantID, terr := obsidian.GetTenantID(c)
+	if terr != nil {
+		return terr
+	}
+
+	_, err := tenants.GetTenant(c.Request().Context(), tenantID)
+	err = errorHandlingForGet(err, fmt.Errorf("Tenant %d does not exist", tenantID), err)
+	if err != nil {
+		return err
+	}
```

Comment:
Nit: here and below, we could refactor this shared functionality out to `validateTenantExists` function